### PR TITLE
[#12422] Can't navigate to community channel from community invite view

### LINKED ISSUE #12422: Can't navigate to community channel from community invite view
# Bug Report

## Problem 
If accessing community view via invite, not possible to navigate to chat as a member of the community

Recent regression, reproducible from 03.08.21, most likely from https://github.com/status-im/status-react/commit/d6d4b28e1eed8351a7183f150ef2c19134e4996e

Blocks 695842 e2e.

#### Expected behavior
can access public chats

#### Actual behavior
https://user-images.githubusercontent.com/4557972/128216912-aab51882-054a-4131-8601-8a7124da05b2.mp4

### Reproduction

1) User A: create community with channel, post link to public channel
2) User B: join public chat, wait until community link will be unfurled, tap on it and request access to community (do not navigate to home view)
3) User A: approve membership request from User A
4) User B: try to access channel 

### Additional Information

- Status version: nightly 04/08/2021
- Operating System: Android, iOS

cc @flexsurfer

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #12422

### CODE REVIEW

**Comment by churik** [comment]
It is navigated, but another chat view is not opened, so you need to go back to open desired chat

https://user-images.githubusercontent.com/4557972/128512488-a50e616e-0fd2-4938-925a-8d0132d63847.mp4

File: src/status_im/ui/screens/communities/community.cljs [hunk 1]
```diff
@@ -139,21 +139,21 @@
...
-(defn community-chat-item [{:keys [chat-id] :as home-item}]
+(defn community-chat-item [{:keys [chat-id] :as home-item} {:keys [from-chat]}]
   [inner-item/home-list-item
    ;; We want communities to behave as public chats when it comes to
    ;; unread indicator
    (assoc home-item :public? true)
    {:on-press      (fn []
                      (re-frame/dispatch [:dismiss-keyboard])
-                     (re-frame/dispatch [:chat.ui/navigate-to-chat chat-id true])
+                     (re-frame/dispatch [:chat.ui/navigate-to-chat chat-id (if from-chat false true)])
```
Comment (shivekkhurana) [inline src/status_im/ui/screens/communities/community.cljs]: `(not from-chat)`

**Review by shivekkhurana — APPROVED** [review]
Looks good

**Comment by flexsurfer** [comment]
thanks @churik fixed

**Review by churik — APPROVED** [review]

**Comment by churik** [comment]
@flexsurfer 
thanks a lot for fixes.
Please, mind my commit while squashing

**Pull request merged**